docker-compose.yml

```diff
@@ -791,7 +799,7 @@ services:
       - OPR_PROXY_HEALTHCHECK=enabled
 
   mariadb:
-    image: mariadb:10.7 # fix issues when upgrading using: mysql_upgrade -u root -p
+    image: mysql:8.0.33 # fix issues when upgrading using: mysql_upgrade -u root -p
```

Comment:
We should revert back to mariadb:10.7, right?

Reply:
From https://github.com/appwrite/docker-mariadb/pull/20 I thought LTS versions where going be the go. `mariadb:lts` is a simple option to pin that there.

`MARIADB_AUTO_UPGRADE=1` might solve whatever `mysql_upgrade` problem you where having.

Also see MariaDB names in: https://github.com/appwrite/docker-mariadb/pull/23